scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/DefaultOpEnvironment.java

```diff
@@ -724,6 +754,13 @@ private synchronized void initIdDirectory() {
 				.forEach(info -> idDirectory.put(info.id(), info));
 	}
 
+	/**
+	 * Function for populating opDirectory with new ArrayList.
+	 * Declared only once, right here
+	 */
+	private final Function<String, List<OpInfo>> nameListCreator = //
+		(name) -> new ArrayList<>();
```

Comment:
I think the parentheses around the `name` parameter are unneeded.

Reply:
Also: is it actually needed to have this function, for performance? Does Ops behave in a worse way if you just inline this lambda below where the `nameListCreator` is passed to `computeIfAbsent`? I'm skeptical that the JIT doesn't just deal with it...

Reply:
Probably not - inlined in 63a1210a